packages/adapters-library/src/scripts/buildMetadataDb.ts

```diff
@@ -155,6 +156,31 @@ async function buildAdapterMetadata(adapter: IProtocolAdapter) {
     return
   }
 
+  const invalidPoolAddress = metadataDetails.find((pool) => {
+    if (pool.address === ZERO_ADDRESS) {
+      return true
+    }
+  })
+
+  if (invalidPoolAddress) {
+    console.error(chalk.yellow(ZERO_ADDRESS))
+
+    console.error(
+      chalk.red(
+        '\n * The above addresses found in the metadata file cannot be a pool address.',
+      ),
+    )
+    console.error(
+      chalk.green('\n * Please ensure that address pool address is correct'),
+    )
+    console.error(
+      chalk.green(
+        '\n * Please checksum your addresses inside the buildMetadata() method.',
+      ),
+    )
+    return
+  }
+
```

Comment:
ensure we dont try to save a pool with the zero address

Reply:
We didn't we add the ZERO_ADDRESS to the method that checks invalid addresses?